packages/block-library/src/gallery/deprecated.js

```diff
@@ -19,844 +19,824 @@ import {
 } from './constants';
 
 const deprecated = [
-	{
```

Comment:
Just temporarily commented out until we finalise new structure and set up new deprecations.